src/main/java/edu/wpi/first/pathweaver/Game.java

```diff
@@ -13,6 +13,7 @@
 
 public final class Game {
   private static final Set<Game> GAMES = new LinkedHashSet<>();
+  public static final Game INFINTE_RECHARGE = loadGameFromResource("2020-infiniterecharge.json");
```

Comment:
Just for the sake of consistency, this should probably be named `INFINITE_RECHARGE_2020`